django_socio_grpc/servicer_proxy.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+
+import grpc
+from django import db
+
+from django_socio_grpc.exceptions import GRPCException
+from django_socio_grpc.request_transformer.grpc_socio_proxy_context import (
+    GRPCSocioProxyContext,
+)
+
+logger = logging.getLogger("django_socio_grpc")
+
+
+def not_implemented(request, context):
+    """Method not implemented"""
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details("Method not implemented!")
+    raise NotImplementedError("Method not implemented!")
+
+
+class ServicerProxy:
+    def __init__(self, ServiceClass, **initkwargs):
+        self.service_instance = ServiceClass(**initkwargs)
+        self.grpc_async = os.environ.get("GRPC_ASYNC")
+
+    def call_handler(self, action):
+        if self.grpc_async:
+            print("async_handler")
```

Comment:
(print) use logger instead or to remove?

Reply:
removed in the next PR. Thanks